src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/entity/BulkPlan.kt

```diff
@@ -47,6 +51,38 @@ class BulkPlan(
   val people: MutableSet<PersonSummary> = mutableSetOf()
 }
 
-interface BulkPlanRepository : JpaRepository<BulkPlan, UUID>
+interface BulkPlanRepository : JpaRepository<BulkPlan, UUID> {
+  @Query(
+    """
+    with existing_alerts as (select a.id as id, a.prisonNumber as pn, a.activeFrom as activeFrom, a.activeTo as activeTo
+                         from Alert a
+                         where a.alertCode.code = :alertCode),
```

Comment:
Does this need to filter out inactive alerts? It looks like this will include those in the update count. The previous logic should have only updated alerts if they had a future dated activeTo value. It set it to null in that case